src/components/TicketsSegment.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+    <div class="ticket__segment segment">
+        <div class="segment__cell">
+            <div class="segment__route segment--title">
+                {{segment.origin}}-{{segment.destination}}
+            </div>
+            <div class="segment__timestart-timeend">
+                {{timeStartEnd}}
+            </div>
+        </div>
+        <div class="segment__cell">
+            <div class="segment--title">
+                В пути
+            </div>
+            <div class="segment__duration">
+                {{durationFormatted}}
+            </div>
+        </div>
+        <div class="segment__cell">
+            <div class="segment__stops-count segment--title">
+                {{stops}}
+            </div>
+            <div class="segment__stops-list">
+                {{segmentStops}}
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    export default {
+        props: {
+            segment: {
+                type: Object,
+                require: true,
+            }
+        },
+        computed: {
+            stops() {
+                if (this.segment.stops.length === 0)
+                    return 'без пересадок'
+
+                return this.segment.stops.length + (this.segment.stops.length === 1 ? ' пересадка' : ' пересадки')
+            },
+            segmentStops() {
+                return this.segment.stops.join(', ')
+            },
+            durationFormatted() {
+                let minutes = Math.floor((this.segment.duration / (1000 * 60)) % 60)
+                let hours = Math.floor((this.segment.duration / (1000 * 60 * 60)) % 24)
+                return hours + 'Ч ' + minutes + 'М'
```

Comment:
Можно было перевести duration в Date и использовать нативное API:
```suggestion
                Intl.DateTimeFormat('ru-RU', { hour: '2-digit', minute: '2-digit' }).format(date)
```